sig-docs/annual-report-2022.md

```diff
@@ -105,3 +106,102 @@ Operational tasks in [sig-governance.md]:
 [sigs.yaml]: https://git.k8s.io/community/sigs.yaml
 [contributor guide]: https://git.k8s.io/community/contributors/guide/README.md
 [devel]: https://git.k8s.io/community/contributors/devel/README.md
+=======
+# **2022 Annual Report: SIG Docs**
+
+***Current initiatives***
+
+What work did the SIG do this year that should be highlighted?
+
+* Contributors across many SIGs, companies, and foundations worked together with SIG Docs to finalize the [dockershim removal](https://kubernetes.io/blog/2022/02/17/dockershim-faq/) from the Kubernetes website
+* Our Localization subproject was officially documented in our community resources, with two subproject leads appointed and the `area/localization` label created
+* The Hindi localization, our first localized set of documentation using the Devanagari script, officially launched
+* A new [Issue Wrangler](https://github.com/kubernetes/website/discussions/38861) role, as a compliment to the [SIG Docs PR Wrangler](https://github.com/kubernetes/website/wiki/PR-Wranglers) role, has been put together to tackle issue triage and feature requests more efficiently
```

Comment:
This hasn't been formalized yet with documentation, so I think this should move to the next section.